pages/docs/pagination.en-US.mdx

```diff
@@ -314,6 +315,33 @@ const getKey = (pageIndex, previousPageData) => {
 }
 ```
 
+### Parallel Fetching Mode
```

Comment:
Consider adding a note that it's only available in >= 2.1.0?

Reply:
That makes sense 👍  I'll add it.

Reply:
done at https://github.com/vercel/swr-site/pull/446/commits/beed70b80e9f9f2efc50ea22a307057789f605db